src/liburing-ffi.map

```diff
@@ -0,0 +1,180 @@
+LIBURING_2.0 {
```

Comment:
Why not just make this for 2.4 and onwards? The older release are not being redone, so not sure why we'd want to have anything but 2.4?

Reply:
You kinda caught me here, I must admit.

I've actually never written a version script before. To get something working, I simply copied what already existed.

You are correct, any version before 2.4 is nonsensical in this instance.